packages/ic-asset-certification/src/asset_router.rs

```diff
@@ -1407,6 +2241,34 @@ mod tests {
         b"<html><body><h1>404 Not Found!</h1></body></html>".to_vec()
     }
 
+    #[fixture]
+    fn not_found_html_gz_body() -> Vec<u8> {
+        vec![
+            49, 19, 9, 8, 0, 0, 0, 0, 0, 0, 3, 23, 9, 32, 1, 64, 32, 1, 32, 5, 23, 7, 23, 9, 115,
+            32, 2, 121, 22, 9, 24, 0, 23, 9, 32, 1, 72, 24, 0, 81, 73, 72, 129, 36, 0, 32, 3, 71,
+            129, 17, 2, 32, 3, 71, 32, 5, 117, 129, 24, 0, 32, 9, 7, 16, 33, 7, 35, 2, 103, 16, 0,
+            36, 5, 25, 3, 116, 1, 37, 4, 36, 9, 17, 0, 114, 73, 0, 0, 0,
+        ]
+    }
+
+    #[fixture]
+    fn not_found_html_zz_body() -> Vec<u8> {
+        vec![
+            120, 156, 179, 201, 40, 201, 205, 177, 179, 73, 202, 79, 169, 180, 179, 201, 48, 180,
+            51, 49, 48, 81, 240, 203, 47, 81, 112, 203, 47, 205, 75, 81, 180, 209, 7, 10, 217, 232,
+            67, 100, 245, 193, 74, 1, 133, 210, 15, 136,
+        ]
+    }
+
+    #[fixture]
+    fn not_found_html_br_body() -> Vec<u8> {
+        vec![
+            27, 48, 0, 248, 45, 14, 108, 27, 88, 249, 245, 45, 213, 3, 233, 193, 146, 199, 9, 173,
+            64, 104, 10, 230, 173, 67, 124, 216, 218, 84, 12, 93, 47, 66, 139, 48, 3, 233, 78, 128,
+            105, 198, 36, 242, 83, 62, 179, 122, 129, 33, 16, 12,
+        ]
```

Comment:
nit: please add a comment what these vectors contain and/or how they were produced.

Reply:
Adding in the next commit, thanks